### DISCUSSION TITLE
NPP 51.2v 280AH (FLCD-30048 or NSFG280F10/DS from the sticker)

### DISCUSSION THREAD

**Discussion by ClarkAlmazan** [syssi/esphome-pace-bms#60] [General]
BMS is **P16S200A-21452-2.02**, using PACE_MODBUS protocol. The battery's touch interface also allows you to change the protocol w/o using pbmstools. 

Every setting/value seems to be visible from the esp32 template, but I haven't tried to change any of the settings.

I used the following for setup:
- Wemos D1 Mini 32 (TX: GPIO17, RX: GPIO16, flow control pin: GPIO15)
- MAX485 w/ flow control module
- ESPHome 2023.11.06

**Comment by syssi** [comment]
I've added your battery bank to the list of supported devices at the README.
